crates/cxx-qt-gen/src/generator/cpp/threading.rs

```diff
@@ -19,10 +19,10 @@ pub fn generate(qobject_idents: &QObjectName) -> Result<GeneratedCppQObjectBlock
     result.forward_declares.push(format!(
         "using {cxx_qt_thread_ident} = ::rust::cxxqtlib1::CxxQtThread<{ident}>;"
     ));
-    result.members.push(CppFragment::Pair {
-        header: format!("::std::shared_ptr<::rust::cxxqtlib1::CxxQtGuardedPointer<{ident}>> m_cxxQtThreadObj;"),
-        source: format!(", m_cxxQtThreadObj(::std::make_shared<::rust::cxxqtlib1::CxxQtGuardedPointer<{ident}>>(this))"),
-    });
+    result.members.push(format!(
+        "::std::shared_ptr<::rust::cxxqtlib1::CxxQtGuardedPointer<{ident}>> m_cxxQtThreadObj;"
+    ));
+    let member_initializer = format!("m_cxxQtThreadObj(::std::make_shared<::rust::cxxqtlib1::CxxQtGuardedPointer<{ident}>>(this))");
```

Comment:
could put this after all the `result.***.push(..)` blocks so it's not hidden  in the middle.

Reply:
Done